openstf/tasks/create_components_forecast.py

```diff
@@ -29,21 +29,78 @@
 
 
 """
-from openstf.pipeline.create_forecast import make_components_prediction
+import structlog
+from datetime import datetime, timedelta
+
+from openstf.pipeline.create_component_forecast import (
+    create_components_forecast_pipeline,
+)
 from openstf.tasks.utils.utils import check_status_change, update_status_change
 from openstf.tasks.utils.predictionjobloop import PredictionJobLoop
 from openstf.tasks.utils.taskcontext import TaskContext
 
+T_BEHIND_DAYS = 0
+T_AHEAD_DAYS = 3
+
 
-def create_components_forecast_pj(pj, context):
+def create_components_forecast_task(pj, context):
```

Comment:
This seems inconsistent with the task above. `create_basecase_forecast` Perhaps it makes sense to postfix the task function with `_task`. But I would suggest to apply that to every task or not.